src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -498,10 +498,10 @@ def getServiceURL(String serviceName, String namespace = null, String protocol =
 }
 
 def isOpenShiftS2I() {
-    def openshiftYaml = "target/classes/META-INF/fabric8/openshift.yml"
+  def openshiftYaml = findFiles(glob: '**/openshift.yml')
```

Comment:
What about if this finds multiple files? Should they all be checked or just the first one as you're doing below?